examples/bert/run_glue_finetuning.py

```diff
@@ -215,15 +230,29 @@ def preprocess_data(inputs, labels):
         hidden_size=bert_config["hidden_size"],
         num_classes=3 if FLAGS.task_name in ("mnli", "ax") else 2,
     )
-    finetuning_model.compile(
-        optimizer=keras.optimizers.Adam(learning_rate=FLAGS.learning_rate),
-        loss="sparse_categorical_crossentropy",
-        metrics=["accuracy"],
-    )
-    finetuning_model.fit(
-        train_ds, epochs=FLAGS.epochs, validation_data=validation_ds
+
+    hypermodel = BertHyperModel(finetuning_model)
+
+    # Initialize the random search over the 4 learning rate parameters, for 4
+    # trials and 3 epochs for each trial.
+    tuner = kt.RandomSearch(
+        hypermodel=hypermodel,
+        objective="val_accuracy",
+        max_trials=4,
+        overwrite=True,
+        directory="tuning_hp_bert",
```

Comment:
Where does this get written? Should we make it configurable where this is stored?

Reply:
It would basically make a new directory for the purpose in the parent directory. Yeah, I suppose we can make it configurable, we can add another argument at the top.

Reply:
I've not yet made changes for this comment, I wanted to ask if we want two arguments, like one for the `directory` and one for the `project_name` as well?

Reply:
@mattdangerw What is the final thing to be done here, should we make two different arguments for `directory` and `project_name`? Or should we just use a default `project_name` value and just make the `directory` name customizable?

Reply:
I guess because the output model will be saved regardless, we won't really need anything out of this working directory for tuner right? Can we just pass a temp directory to directory?

Reply:
Yeah, the directory where the hyperparameter-tuning related stuff will be stored, won't be used(unless the user wants to) so I suppose we can just make a temp directory name, for example, leave it as the current one `tuning_hp_bert` or something more readable like `hyperparameter_tuner_results`?